test/bigint.jl

```diff
@@ -432,3 +432,8 @@ end
     y = rand(UInt128)
     @test BigInt(y) % UInt128 === y
 end
+
+@testset "conversion from typemin($T)" for T in (Int8, Int16, Int32, Int64, Int128)
+    x = typemin(T)
+    @test big(x) == x
```

Comment:
Is this actually that useful of a test? If the comparison uses promotion, then it won't work correctly.

Reply:
True. Currently, comparison does not promote (that's how I stumbled upon this), but if it does in the future, this test will become useless. I'll change to use explicit constants.